ports/opentxs/vcpkg.json

```diff
@@ -1,18 +1,21 @@
 {
     "name": "opentxs",
     "version": "1.203.2",
-    "port-version": 0,
+    "port-version": 3,
     "description":
         "The Open-Transactions project is a collaborative effort to develop a robust, commercial-grade, fully-featured, free-software toolkit implementing the OTX protocol as well as a full-strength financial cryptography library, API, CLI, and prototype server. The project is managed by a worldwide community of volunteers that use the Internet to communicate, plan, and develop the Open-Transactions toolkit and its related documentation.",
     "homepage": "https://opentransactions.org",
     "features": {
         "qt6": {
-            "description": "Link to Qt6",
+            "description": "Qt Qt provided by vcpkg",
```

Comment:
This should say Use Qt...

Reply:
will fix the next time this port is updated.